src/include/cart/api.h

```diff
@@ -1947,6 +1947,65 @@ int crt_group_secondary_create(crt_group_id_t grp_name,
  */
 int crt_group_secondary_destroy(crt_group_t *grp);
 
+
+/**
+ * Perform a primary group modification in an atomic fashion based on the
```

Comment:
in my old project (threadchecker), adding APIs means a minor version bump (e.g. 2.0 to 2.1).  I wonder if that's the same for RPMs.

Reply:
It is.  `rpmbuild` auto-creates dependencies on the major version of any libraries the thing it's packaging needs.  i.e.:

```
$ rpm -q --requires daos
...
libhwloc.so.5()(64bit)
...
``` 

and then 

```
$ rpm -q --whatprovides 'libhwloc.so.5()(64bit)'
hwloc-libs-1.11.8-4.el7.x86_64
```

So as you can see, `rpmbuild` is populating consumers and providers with the right tags to make sure that when you install a consumer you get the right provider without one having to manually specify dependencies in the RPM specfile.  It's really quite elegant.